Python/pywarpx/timestepper.py

```diff
@@ -61,7 +61,7 @@ def onestep(self):
 
         self.cur_time += dt
 
-        libwarpx.warpx_MoveWindow();
+        libwarpx.warpx_MoveWindow(self.istep,false);
```

Comment:
As above for the 2nd argument (`move_j`)

Reply:
Same here, I would use `true`.